src/state/endpoints/tests/selectors.test.js

```diff
@@ -0,0 +1,25 @@
+import { getEndpoints } from '../selectors';
+
+const endpoints = [
+  { path_labeled: 'myEndpoint' }
+];
+
+const state = {
+  endpoints: {
+    wpcom: {
+      v1: endpoints
+    }
+  }
+};
+
+it('should return empty array if unknown apiName', () => {
+  expect(getEndpoints(state, 'wporg', 'v1')).toEqual([]);
+});
+
+it('should return empty array if unknown version', () => {
+  expect(getEndpoints(state, 'wpcom', 'v11')).toEqual([]);
+});
```

Comment:
Looking at implementation, one "fun" gotcha could be this failing test:

```js

it('should return empty array for object proto', () => {
  expect(getRecentEndpoints(state, 'wpcom', 'valueOf')).toEqual([]);
});
```

Might consider `Object#hasOwnProperty` or Lodash's `_.get` instead.